core/src/scss/components/alert/_alert.scss

```diff
@@ -7,51 +7,150 @@
 //
-// .su-alert--success  - Successful Transaction Message
-// .su-alert--warning  - Warning Message
-// .su-alert--error    - Error Message
-// .su-alert--info     - Informative Message
+// .su-alert--success su-alert--text-light    - Successful Transaction Message
+// .su-alert--warning                         - Warning Message
+// .su-alert--error su-alert--text-light      - Error Message
+// .su-alert--info su-alert--text-light       - Informative Message
+// .su-alert--text-light                      - Lighter color text for darker backgrounds
 //
 // Markup: templates/components/alert/alert.twig
 //
 // Style guide: Simple.Alert
 //
 .su-alert {
-  @include padding(2rem);
-  background-color: color(default, $su-alert-colors);
-  background-position: 1rem center;
-  background-repeat: no-repeat;
-  background-size: 4rem;
-  color: color(dark-text, $su-alert-colors);
+  background-color: $su-color-fog;
 
-  @include grid-media('md') {
-    background-size: 5.2rem;
+  .centered-container {
+    @include padding(1em null);
+    display: flex;
+    flex-wrap: wrap;
+
+    @include grid-media("sm") {
+      align-items: center;
+    }
   }
+}
 
-  .su-alert__body {
-    display: table-cell;
-    vertical-align: top;
+// Dismiss close button.
+// -----------------------------------------------------------------------------
+.su-alert__dismiss {
+  @include modular-spacing('margin', null null null 1);
+  order: 3;
+  height: 100%;
+  align-content: flex-end;
+  flex-shrink: 1;
+  text-align: right;
 
-    @include padding(null null null 3.5rem);
+  @include grid-media-max('xs') {
+    width: 100%;
+  }
 
-    @include grid-media('md') {
-      @include padding(null null null 5rem);
-    }
+  .su-alert__dismiss-button {
+    @include padding(0);
+    background-color: transparent;
+    text-transform: uppercase;
+    font-weight: $su-font-semi-bold;
+    color: $su-color-black;
+    font-size: 1.7rem;
 
-    p:last-child {
-      @include margin(null null 0.8rem);
+    .su-far {
+      @include margin(null null null 0.5em);
     }
+  }
+}
 
-    .su-alert__heading {
-      @include margin(0 null 0.3rem);
+// Flex header with the icon and message type label.
+// -----------------------------------------------------------------------------
+.su-alert__header {
+  @include modular-spacing('margin', null 1 null null);
+  order: 1;
+  flex-shrink: 1;
 
-      @include grid-media('md') {
-        @include margin(0.3rem null null);
-      }
+  @include grid-media-max('md') {
+    width: 100%;
+  }
+}
+
+// Icon Styles.
+// ------------
+.su-alert__icon {
+  @include margin(null 0.5em null null);
+  display: inline-block;
+  max-width: 20px;
+
+  i {
+    font-style: normal;
+  }
+}
+
+// Alert Type Styles.
+// ------------------
+.su-alert__label {
+  height: 100%;
+  line-height: 100%;
+  display: inline-block;
+  font-size: 1.7rem;
+  text-transform: uppercase;
+  font-weight: $su-font-semi-bold;
+  margin-bottom: 1em;
```

Comment:
Please see the screenshot in the other comment - `.su-alert__label` will need `margin-bottom: 0;` whether or not there is footer.